cclient/iota_api.c

```diff
@@ -30,6 +30,23 @@ iota_api_result_t iota_api_get_node_info(
 iota_api_result_t iota_api_get_neighbors(
     const iota_http_service_t* const service, get_neighbors_res_t* const res) {
   iota_api_result_t result = {0};
+
+  char_buffer_t* req_buff = char_buffer_new();
+  char_buffer_t* res_buff = char_buffer_new();
```

Comment:
Could you check their return values (oom) ?

Reply:
sure